deploy/docker-compose/docker-compose-local-baseline.yml

```diff
@@ -64,7 +64,9 @@ services:
       - RAILS_HIDE_STACKTRACE=true
       - ADS_PORT=${ADS_PORT}
       - DISCOUNTS_PORT=${DISCOUNTS_PORT}
-    image: "ddtraining/storefront-fixed:latest"
+      - ADS_ROUTE=${ADS_ROUTE}
+      - DISCOUNTS_ROUTE=${DISCOUNTS_ROUTE}
+    image: "public.ecr.aws/x2b9z2t7/ddtraining/storefront-fixed:latest"
```

Comment:
Should this be tagged with the associated release, rather than `latest`?

Reply:
This compose file is used for local development only, so I keep it tagged to `latest` so the most recent changes to storefront are always being tested if we're between releases